File: chapter_002/compare_txt.py
#!/usr/bin/env python3

a_file = "./fizzbuzz_python.txt"
b_file = "./fizzbuzz_rust.txt"

with open(a_file, "r") as f:
    a_lines = f.read()

with open(b_file, "r") as f:
    b_lines = f.read()

a_lines = a_lines.split()
b_lines = b_lines.split()

if a_lines == b_lines:
    print("OK")
else:
    print("NG")

File: chapter_002/compare_txt.rs
use std::fs;

fn main() {
    let a_file = "./fizzbuzz_rust.txt";
    let b_file = "./fizzbuzz_python.txt";

    let a_content = fs::read_to_string(a_file)
                        .expect("Something went wrong reading the file");
    let b_content = fs::read_to_string(b_file)
                        .expect("Something went wrong reading the file");


    if a_content == b_content {
        println!("The files are the same.");
    } else {
        println!("The files are different.");
    }

}
